libnavui-base/src/main/java/com/mapbox/navigation/ui/base/internal/utils/ClickListenersDelegate.kt

```diff
@@ -0,0 +1,27 @@
+package com.mapbox.navigation.ui.base.internal.utils
+
+import android.view.View
+import java.util.HashSet
+
+class ClickListenersDelegate : View.OnClickListener {
```

Comment:
it's used to combine multiple click listeners(user might add a few ClickListener to view), believe we shouldn't restrict API, because some customers might expect this approach here as well as in 1.x

Reply:
What do you mean by `user might add a few ClickListener to view`? Do you mean more than one click listeners? I don't know the motivation for this approach in `1.x`, but looking at the use case it's pretty simple that you toggle the state of the button when you click on it. We also haven't seen any kind of different requests from the customers. So, I don't think this is necessary at the moment.

I would rather suggest to expose the FAB using a `getSoundButton()` API, so that gives access to FAB and you can set on click listener to it if need be.